src/main/java/com/eternalcode/discordapp/command/EmbedCommand.java

```diff
@@ -0,0 +1,66 @@
+package com.eternalcode.discordapp.command;
+
+import com.eternalcode.discordapp.Embeds;
+import com.freya02.botcommands.api.annotations.Optional;
+import com.freya02.botcommands.api.annotations.UserPermissions;
+import com.freya02.botcommands.api.application.ApplicationCommand;
+import com.freya02.botcommands.api.application.annotations.AppOption;
+import com.freya02.botcommands.api.application.slash.GuildSlashEvent;
+import com.freya02.botcommands.api.application.slash.annotations.JDASlashCommand;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import org.jetbrains.annotations.NotNull;
+
+import java.awt.*;
+
+@UserPermissions(Permission.MANAGE_CHANNEL)
+public class EmbedCommand extends ApplicationCommand {
+
+    @JDASlashCommand(
+            name = "embed",
+            description = "Send embed message"
+    )
+    public void onSlashCommand(
+            @NotNull GuildSlashEvent event,
+            @NotNull @AppOption(name = "title") String title,
+            @NotNull @AppOption(name = "description") String description,
+            @Optional @AppOption(name = "color") String color,
+            @Optional @AppOption(name = "footer") String footer,
+            @Optional @AppOption(name = "author") String author,
+            @Optional @AppOption(name = "thumbnail-url") String thumbnailImage
+    ) {
+        try {
+            EmbedBuilder embed = new EmbedBuilder();
+            embed.setTitle(title);
+            embed.setDescription(description);
+            embed.setColor(Color.decode(color != null ? color : "#FFFFFF"));
+            embed.setFooter(footer != null ? footer : "");
+            embed.setAuthor(author != null ? author : "");
+            embed.setThumbnail(thumbnailImage);
+
+            MessageEmbed messageEmbed = embed.build();
+
+            MessageEmbed successSend = new Embeds().success
+                    .setDescription("Embed sent successfully")
```

Comment:
```suggestion
                    .setDescription("✅ | Embed sent successfully")
```

Reply:
Brzydko wygląda

Reply:
W Embedzie właśnie to ładnie będzie wyglądało IMO